badge/badge.css

```diff
@@ -0,0 +1,62 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Montserrat&display=swap');
+@import url('../styles.css');
+@import url('../avatar/avatar.css');
+
+html,body{
+    font-family: 'Montserrat', sans-serif;
+}
+
+.badge{
+    border : 1px solid transparent;
+    padding: 0.6rem 1rem;
+    background-color: var(--primary-white);
+    margin: 0.5rem;
+    /* border: 2px solid black; */
+}
+.badge__text{
```

Comment:
This naming scheme isn't consistent with Alert styles. There you used "-" here "__" is used. Keeping it common across the whole project.

Reply:
I modified most of the parts to use one naming convention, thanks a lot for pointing out that.